racingcar-java/src/main/java/racingcar/ResultView.java

```diff
@@ -0,0 +1,34 @@
+package racingcar;
+
+public class ResultView {
+
+    private static char printCharacter = '-';
```

Comment:
```suggestion
    private static final char printCharacter = '-';
```
의미있는 상수 추출 👍
상수는 `static final`을 붙여사용합니다